src/js/components/homepage/visualizations/treemap/TreeMapCell.jsx

```diff
@@ -127,25 +181,31 @@ export default class TreeMapCell extends React.Component {
         return (
             <g
```

Comment:
I think there needs to be a cursor icon change to indicate when these cells are clickable.